### ISSUE TITLE
a11y concerns: Missing visual focus indicator, elements not accessible by the Keyboard

### ISSUE THREAD

**Issue by greydnls** [filestack/filestack-react#108]
Hi, I'm using version 3.2 of filestack-react, with this configuration: 

```js
const options = {
    onSuccess,
    onError,
    apikey: FILEPICKER_API_KEY,
    componentDisplayMode: {
      type: 'immediate',
    },
    actionOptions: {
      container,
      onUploadStarted,
      accept: ['.pdf', '.doc', '.docx'],
      displayMode: 'inline',
      fromSources: ['local_file_system', 'url', 'github', 'googledrive'],
      lang: LANGUAGE,
      storeTo: { workflows: [FILEPICKER_WORKFLOW_ID] },
    },
  };

  return <ReactFilestack {...options} />;
```

This renders the picker, and works as expected, generally. There are a few accessibility errors we're noticing, though. I don't see any way to override this behavior, so I thought I would open up an issue.  

1.  None of the buttons highlighted in the image below are accessible by keyboard. Additionally, they are lacking labels and in some cases roles, which makes them hard to use with a screen reader. 
<img width="836" alt="Screen Shot 2021-01-12 at 3 00 55 PM" src="https://user-images.githubusercontent.com/1276798/104366558-8f1aaa00-54e7-11eb-9379-6f98998c3224.png">

2. The dropzone itself never registers as being "active", when you tab into it. 
This: 
![image](https://user-images.githubusercontent.com/1276798/104365735-5cbc7d00-54e6-11eb-868a-cd4c09236042.png)
Versus the expected (achieved by manually giving that element focus): 
![image](https://user-images.githubusercontent.com/1276798/104365840-82498680-54e6-11eb-9d12-45eedc67c0af.png)

3. The file upload input item doesn't have a valid label. Possible solutions, fix any of the following: 
- provide aria-label attribute 
- provide aria-labelledby attribute that references a valid element
- Add an implicit (wrapped) <label>
- Add an explicit <label>
- add title attribute 

Thanks for all your work on this tool, and thanks in advance for considering this issue. Please let me know if you need more information or if there's something I've missed in my setup.

**Comment by sarahbilly** [comment]
I would also love to see this resolved!

**Comment by jjun14** [comment]
Bumping this issue, would love to see this addressed soon 🙏
